js/script.js

```diff
@@ -27,11 +25,93 @@ var playerMove = function(move) {
   } else {
     aiMove = 'scissors';
   }
+}
+
+var printResult = function(userMove) {
```

Comment:
userMove is in global variables, so you should decide from where you want to take value (better from arguments, but then you need to rebuild your code and remove global)